yastd/dbus_api.md

```diff
@@ -0,0 +1,169 @@
+
+## get the list of available languages
+
+Iface: o.o.YaST.Installer1.Language
+
+methods:
+
+  ToInstall(array(string)) -> void
+    Set list of languages to install
+    Example:
+      ToInstall(["cs_CZ", "de_DE"]) -> () # only lang codes from AvailableLanguages is supported
+
+Properties (all read only):
+
+  AvailableLanguages -> array(array(string, string, dict(string, variant)))
+    List of all available languages to install on target system.
+    Example:
+      AvailableLanguages -> [["cs_CZ", "Czech", {}]] # it is lang code, human readable lang name and dict for future extensions to provide more data
+
+  MarkedForInstall -> array(string)
```

Comment:
Marked or Selected?

Reply:
it really depends on installer behavior. If all marked for install will be used or not.

Reply:
OK, I see. It is just that I saw `SelectedBaseProduct` and I thought about using the same pattern.